app/pages/feedbacks_list.php

```diff
@@ -0,0 +1,52 @@
+<?php
+restrict_access([Permission::ROOT]);
+
+if ($_SERVER['REQUEST_METHOD'] === 'DELETE') {
+    // Extract the ID from the URL
+    $id = basename($_SERVER['REQUEST_URI']);
```

Comment:
We have a better way of getting the ID here
```suggestion
    $id = get_route_param("feedback_id");
```
This will ensure that `$id` is numeric and not null.

Reply:
Done